### ISSUE TITLE
Sequence or flow diagram

### ISSUE THREAD

**Issue by abhi007tyagi** [actions-on-google/apiai-transactions-nodejs#11]
Hi,

Please add a sequence diagram or a flow diagram with intent names and responses which can easily depict how the communication is happening between hardware, Google assistant, webhook and third-party server (for payment and reservation). This way it will be clearly understandable for anyone to see how to implement it in their flow.

**Comment by xtraad** [comment]
This is useful:
https://www.eisbahn.jp/yoichiro/2018/04/explanation_of_transactions_api.html

**Comment by taycaldwell** [comment]
The blog provided above by one of our GDEs has a very nice diagram explaining the flow.

If you find the code hard to follow, I recommend first reading our [Transactions Documentation](https://developers.google.com/actions/transactions/) to get a better idea on how to implement transactions for your action.
